spec/requests/posts_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe PostsController, type: :request do
+  describe 'GET /posts' do
+    before(:example) { get '/users/1/posts' }
+    it 'should check the responce is comming' do
+      expect(response).to have_http_status(:ok)
+    end
+    it('is correct template rendered') do
+      expect(response).to render_template(:index)
+    end
+  end
+  describe 'GET /posts/:id' do
+    before(:example) { get '/users/1/posts/1' }
+    it 'should check the responce is comming' do
+      expect(response).to have_http_status(:ok)
+    end
+    it('is correct template rendered') do
+      expect(response).to render_template(:show)
+    end
+    it('is response body includes correct placeholder text') do
+      expect(response.body).to include('Show the detail of the post')
+    end
+  end
```

Comment:
- [ ] Only 4 of your 10 request tests are passing. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/54848363/229886196-a78e667a-4c06-4e7a-be17-2c1968a60e35.png)

  You may need to create the users and posts here in the test file:

  ![image](https://user-images.githubusercontent.com/54848363/229886576-0306e4f4-bd4f-425b-8354-17317411b135.png)

Reply:
Hello @DammyShittu, How are you 
I just wanted to ask. Have you run  `bundle install` before running the test 
Because at my side all the test is passing 
![image](https://user-images.githubusercontent.com/80938128/229900486-f9299554-2c9e-4b57-a6d1-e627bf10f64b.png)